### ISSUE TITLE
Framework / Typography

### ISSUE THREAD

**Issue by titouanmathis** [studiometa/scss-toolkit#13]
Global improvements:

- [x] Add keys to the `font-sizes` map
- [x] Create a new map `font-display` with all font styles : size, line-height, weight, letter-spacing, family, etc.

**Comment by notjb** [comment]
$font-faces update

```scss
$font-faces: (
  (
    name: 'Meta OT',
    filename: 'meta-ot-regular',
    weight: 400,
    style: normal,
  ),
  (
    name: 'Meta OT',
    filename: 'meta-ot-regular-italic',
    weight: 400,
    style: italic,
  )
);
```

**Comment by titouanmathis** [comment]
Proposal in #23

**Comment by titouanmathis** [comment]
Fixed in #23
